tests/test_task.py

```diff
@@ -341,3 +341,9 @@ def test_task_template_has_correct_labels(op):
     tt = t.get_task_template()
     expected_labels = {'foo': 'bar'}
     assert tt.metadata.labels == expected_labels
+
+
+def test_task_set_as_privaliged(no_op):
```

Comment:
My meaningless feedback: 
```suggestion
def test_task_set_as_privileged(no_op):
```

🙂